recipe/meta.yaml

```diff
@@ -38,7 +38,7 @@ requirements:
     # Capping `lightning` does not cap `pytorch_lightning`, so we cap manually
     - pytorch-lightning >=2.0,<2.2
     - scipy >=1.7.3,<1.8  # [py<38]
```

Comment:
I think this could be fixed too?

```suggestion
    - scipy >=1.7.3,<2  # [py<38]
```

Reply:
My reasoning for this was that `~=1.7.3` is equivalent to `>=1.7.3, ==1.7.*` such that `1.8` would be excluded. I based this on this [document](https://packaging.python.org/en/latest/specifications/version-specifiers/#compatible-release). I'm not certain it's the right source to use though.